scripts/generate-a3p-submission.sh

```diff
@@ -3,17 +3,22 @@ set -ueo pipefail
 
 sdkroot=$(cd -- "$(dirname "$0")/.." >/dev/null && pwd)
 
-cd $sdkroot
+cd "$sdkroot"
 
 buildSubmission() {
-  proposalName=$1
-  a3pProposal=$2
+  proposalName=$1; shift
```

Comment:
why do you prefer shift to ordinals? I see it's more flexible, but it seems like the positions should be maintained